app/src/test/java/org/oppia/app/testing/ProfileChooserSpanTest.kt

```diff
@@ -0,0 +1,147 @@
+package org.oppia.app.testing
+
+import android.content.Context
+import android.content.res.Configuration
+import android.view.View
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import androidx.test.core.app.ActivityScenario.launch
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.intent.Intents
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.profile.ProfileChooserFragment
+import org.robolectric.annotation.Config
+
+private const val TAG_PROFILE_CHOOSER_FRAGMENT_RECYCLERVIEW = "profile_recycler_view"
```

Comment:
Nit: Should be ``TAG_PROFILE_CHOOSER_FRAGMENT_RECYCLER_VIEW`` (recycler view as two words) per https://developer.android.com/kotlin/style-guide#camel_case.

Reply:
Removed